### ISSUE TITLE
Missing emoji icon when Show Send Message button is enabled

### ISSUE THREAD

**Issue by win98se** [TakosThings/Fluent-Discord#127]
**Describe the bug**
Emoji icon goes missing when the Show Send Message button is enabled in Accessibility options; That transparent button can still be clicked normally.

**Screenshots**
![image](https://user-images.githubusercontent.com/11819816/167091437-5823b89d-75cc-4f40-b48b-684a74b093f5.png)
![image](https://user-images.githubusercontent.com/11819816/167091801-905b2f90-93e6-408e-859d-3c6764c65e4e.png)

**Comment by TakosThings** [comment]
Related to #74

Best that can be done is to turn off the send button.

**Comment by win98se** [comment]
![image](https://user-images.githubusercontent.com/11819816/177079482-d29aff0c-377e-4cbc-9f8d-6df6eb062bd4.png)

Fixed in #136 - thank you.
